cli/src/pcluster/models/cluster.py

```diff
@@ -627,32 +639,14 @@ def get_running_capacity(self, updated_value: bool = False):
     def start(self):
         """Start the cluster."""
         try:
+            stack_status = self.stack.status
+            if "FAILED" in stack_status:
+                LOGGER.warning("Cluster stack is in %s status. This operation might fail.", stack_status)
```

Comment:
shall we always raise a BadRequestClusterActionError at the beginning of start and stop if stack status is not in a complete state?

Reply:
Done